### ISSUE TITLE
[Suggestion] Add a option for folders

### ISSUE THREAD

**Issue by ShatteredxEmotions** [lap00zza/9anime-Companion#112]
**Select your issue type:** (check at least one)
- [ ] Bug
- [ ] Question
- [ ] Suggestion
- [ ] Other (please describe):

**Describe your issue:**

**Comment by Vysair** [comment]
Yeah great suggestion, this is quite the pain to separate 3 different anime with 20/12 episode each in 9anime companion folder.

**Comment by lap00zza** [comment]
Please take a look at this comment once:
* https://github.com/lap00zza/9anime-Companion/issues/52#issuecomment-326259523
